### ISSUE TITLE
Cannot Edit Connection-Server

### ISSUE THREAD

**Issue by noxare** [rustdesk/rustdesk#578]
The window with the connection server close instantly when i open it.

...

**What did you expect to happen instead?**
I can entry my own server.

...

**How did you install `RustDesk`?**

--- 1.1.9 portable

**RustDesk version and environment**

 windows 10

I think the Problem is that our firewall (managed from manufacturer) blocks the Rustdesk network direktly and Rustdesk trys to poll this the whole time, even when i disable the connectionserver in the options.

With best regards

O. Krämer

**Comment by rustdesk** [comment]
Noted, Thanks

**Comment by brunoramos89** [comment]
Hello, I am testing Rustdesk at my institution and chose to use my own self-hosting server. I noticed the same problem, when I try to edit the server ID/Relay the screen closes immediately, as if the program was trying to update itself, not allowing me to configure the local server. I believe the reason is that my firewall is blocking the connection to the public servers, so it can't get the ID. 

A suggested solution is: If the connection to the public servers fails, the program displays a message and gives the option to connect to a local server address or try to connect again to the public server (maybe inform the ports 21115, 21116, 21117, 21118, 21119 that need to be released in the firewall).

I hope I have contributed to the evolution of the software. Thank you very much and congratulations on a great tool.
